root/static/scripts/edit/URLCleanup.js

```diff
@@ -821,6 +835,15 @@ validationRules[LINK_TYPES.bbcmusic.artist] = function (url) {
   return /bbc\.co\.uk\/music\/artists\//.test(url);
 };
 
+// allow only BookBrainz entity pages with the BookBrainz rel
+function validateBookBrainz(url) {
+  return /bookbrainz\.org\/[^\/]*\/[a-f0-9]{8}-[a-f0-9]{4}-[a-f0-9]{4}-[a-f0-9]{4}-[a-f0-9]{12}/.test(url);
```

Comment:
That was in my first draft, but I removed it as it is out of the scope of the current issue which is: _to prevent using the BookBrainz relationship with unrelated websites such as Amazon_.

I would be happy to make the validation more precise if there was a note on which BB entities should be allowed in relationships for each MB entity (see the test file for non-exhaustive examples). What about artist’s biography for example? Should it be linked to the publication, to the edition, or to the work? Moreover, there are more BookBrainz entity types to come (any soon?). Maybe it is too early to decide on this additional issue. I will try to gather more information about it.